lightkurve/search.py

```diff
@@ -407,6 +473,34 @@ def search_lightcurvefile(target, radius=None, cadence='long',
         return SearchResult(None)
 
 
+def search_TESScutout(target, sector=None):
```

Comment:
Let's make this a lowercase function name (because all other function names in Lightkurve are lowercase).  How about `search_cutout()`?